### PR TITLE
Added Turkish translation and some other i18n work.

### PR DESCRIPTION
**Pull request by ismailcarlik**

### CODE REVIEW

**Comment by jjw24** [comment]
Hey thanks for the PR, I will take a look later. One thing I noticed though is UWPDesktop nuget package is removed, what's the idea behind that？

**Comment by ismailcarlik** [comment]
> Hey thanks for the PR, I will take a look later. One thing I noticed though is UWPDesktop nuget package is removed, what's the idea behind that？

oh, I forgot adding it back before commit. I am using Windows SDK 17763 and it doesn't require -also fails at build phase- UWPDesktop package.

File: Plugins/Wox.Plugin.BrowserBookmark/Main.cs [hunk 1]
```diff
@@ -48,7 +50,7 @@ public List<Result> Query(Query query)
             return returnList.Select(c => new Result()
             {
                 Title = c.Name,
-                SubTitle = "Bookmark: " + c.Url,
+                SubTitle = context.API.GetTranslation("wox_plugin_browserbookmark_bookmark") + ": " + c.Url,
```
Comment (jjw24) [inline Plugins/Wox.Plugin.BrowserBookmark/Main.cs:53]: This part of the subtitle description("Bookmark") will be removed per https://github.com/jjw24/Wox/pull/51 so translation for this one will not be required

File: Plugins/Wox.Plugin.Program/Wox.Plugin.Program.csproj [hunk 2]
```diff
@@ -61,6 +61,9 @@
     <Reference Include="System.Xaml" />
     <Reference Include="System.Xml" />
     <Reference Include="System.Xml.Linq" />
+    <Reference Include="Windows">
+      <HintPath>$(MSBuildProgramFiles32)\Windows Kits\10\UnionMetadata\10.0.17763.0\Windows.winmd</HintPath>
```
Comment (jjw24) [inline Plugins/Wox.Plugin.Program/Wox.Plugin.Program.csproj:65]: Please reverse this change. Not sure how this is brought in

File: Plugins/Wox.Plugin.Program/Wox.Plugin.Program.csproj [hunk 3]
```diff
@@ -150,13 +158,6 @@
     </ProjectReference>
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="..\..\packages\UwpDesktop.10.0.14393.3\build\portable-net45+uap\UwpDesktop.targets" Condition="Exists('..\..\packages\UwpDesktop.10.0.14393.3\build\portable-net45+uap\UwpDesktop.targets')" />
```
Comment (jjw24) [inline Plugins/Wox.Plugin.Program/Wox.Plugin.Program.csproj]: Please reverse the UWPDesktop Nuget package removable. This is required for Program plugin to work properly. May consider another option in the future as discussed in some PRs in the upstream repo.

File: Plugins/Wox.Plugin.Program/packages.config [hunk 4]
```diff
@@ -3,5 +3,4 @@
   <package id="JetBrains.Annotations" version="10.3.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
   <package id="System.Runtime" version="4.0.0" targetFramework="net452" />
-  <package id="UwpDesktop" version="10.0.14393.3" targetFramework="net452" />
```
Comment (jjw24) [inline Plugins/Wox.Plugin.Program/packages.config:6]: Please reverse the UWPDesktop Nuget package removal. This is required for Program plugin to work properly. 

You can follow the readme instruction to get the build working with UWPDesktop, basically just need to install the specific windows sdk version.

May consider another option in the future as discussed in some issues in the upstream repo.

**Comment by jjw24** [comment]
Hey this is a really nice PR, thank you for your work on the Turkish translation and tidying up the English versions, really like how you separated the keys to dialog and standard sys display.

I would love to merge and release this if you could update the PR with the above comments.

**Comment by ismailcarlik** [comment]
I accidentally removed my fork while trying to remove my wox/Wox repo. So I can't make any changes now.

I suppose you should close this PR without merging and later i'll do the changes again with the points you subjected above.

**Pull request closed without merging**